ios/UniversalAnalyticsPlugin.h

```diff
@@ -6,9 +6,9 @@
 #import "GAI.h"
 
 @interface UniversalAnalyticsPlugin : CDVPlugin {
-    bool _trackerStarted;
     bool _debugMode;
 	NSMutableDictionary *_customDimensions;
+    NSMutableDictionary *_trackers;
```

Comment:
Yes, but we will leave them as protected, like other instance variables in plugin.